js/sum.js

```diff
@@ -0,0 +1,47 @@
+window.addEventListener('DOMContentLoaded', () => {
+    let addBtn = document.getElementById('add-btn');
+    let newNum = document.getElementById('new-num');
+    let numsContainer = document.getElementById('added-nums-container');
+    let sumElem = document.getElementById('sum');
+    let isOdd = false;
+
+    addBtn.addEventListener('click', () => {
```

Comment:
This function is very long and is doing a lot of things at the same time: styling, constructing and calculating. You can try to separate it to smaller functions with clear responsibility. Basically the whole logic is happening in 1 method and it is hard to follow the logic there.